frontend/scripts/metrica.js

```diff
@@ -0,0 +1,231 @@
+(() => {
+    let state = {
+        host: 'http://stage.metrica.fun/api/v1',
+        routes: {
+            create_visitor: '/visitors',
+            create_visit: '/visits'
+        }
+    };
+
+    const getTrackingId = () => {
+        let scripts = document.getElementsByTagName('script');
+        let index = scripts.length - 1;
+        let myScript = scripts[index];
+
+        return (myScript.src.split('tracking_id' + '=')[1] || '').split('&')[0]
+    };
+
+    const createVisitor = (tracking_number) => {
+        return fetch(state.host + state.routes.create_visitor, {
```

Comment:
create over "fetch" some function, because we probably will need to change implementaion from fetch to more generic, unless the babel manages it